app/server/lib/OIDCConfig.ts

```diff
@@ -166,10 +168,10 @@ export class OIDCConfig {
 }
 
 export async function getOIDCLoginSystem(): Promise<GristLoginSystem|undefined> {
-  if (!process.env.GRIST_OIDC_SP_HOST) { return undefined; }
+  if (!process.env.GRIST_OIDC_IDP_ISSUER) { return undefined; }
```

Comment:
Maybe would be useful to mention in top comments that this is the variable required to turn on this login system. And mention also that GRIST_OIDC_SP_HOST is optional.